Change Partner guide to Platform integration guide (#3623)
neondatabase/website: content/docs/guides/consumption-metrics.md

## Consumption polling FAQ

> As a platform integration or paid plan customer, you may have questions related to polling Neon's consumption APIs. We've provided answers to frequently asked questions here.

Reviewer: ```suggestion
As an integrator of Neon or paid plan customer, you may have questions related to polling Neon's consumption APIs. We've provided answers to frequently asked questions here.
```

### How often should consumption data be polled to report usage to customers?

> As mentioned above, usage data can be pulled every 15 minutes, but platform integrations are free to choose their own reporting interval based on their requirements.

Reviewer: ```suggestion
As mentioned above, usage data can be pulled every 15 minutes, but integrators of Neon are free to choose their own reporting interval based on their requirements.
```

### How often should consumption data be polled to invoice end users?

> Neon does not dictate how integrators of Neon bill their users. Platform integrations can use the data retrieved from the consumption API to generate invoices according to their own billing cycles and preferences.

Reviewer: ```suggestion
Neon does not dictate how integrators of Neon bill their users. Integrators of Neon can use the data retrieved from the consumption API to generate invoices according to their own billing cycles and preferences.
```
